tests/backup/backup_test.go

```diff
@@ -922,14 +922,14 @@ var _ = Describe("{ShareBackupWithUsersAndGroups}", func() {
 		Step("Add users to group", func() {
 			log.InfoD("Adding users to groups")
 			var wg sync.WaitGroup
-			for i := 0; i < len(users); i++ {
+			for i, user := range users {
```

Comment:
make this also username to differentiate between user object and username string